test/csuite/CMakeLists.txt

```diff
@@ -89,8 +89,12 @@ define_c_test(
     TARGET test_tiered_abort
     SOURCES tiered_abort/main.c
     DIR_NAME tiered_abort
+    # We need to manually specify the location of the local store library
+    # and build directory as this path is more dynamic compared to layout autoconf
+    # produces in build_posix.
+    FLAGS "-DWT_STORAGE_LIB=\"ext/storage_sources/local_store/libwiredtiger_local_store.so\""
     EXEC_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/tiered_abort/smoke.sh
```

Comment:
or alternatively, I'm trying to move away from calling `smoke.sh` in CMake (for platform portable reasons). Aside from the previous suggestion (dropping the `EXEC_SCRIPT` call), we could instead use the `VARIANTS` feature here to simulate the type of invocations `smoke.sh` makes e.g.
```
define_c_test(
    TARGET test_tiered_abort
    SOURCES tiered_abort/main.c
    DIR_NAME tiered_abort
    # We need to manually specify the location of the local store library
    # and build directory as this path is more dynamic compared to layout autoconf
    # produces in build_posix.
    FLAGS "-DWT_STORAGE_LIB=\"ext/storage_sources/local_store/libwiredtiger_local_store.so\""
    VARIANTS
        "test_tiered_abort_5_thread;-b {CMAKE_BINARY_DIR} -t 10 -T 5"
        "test_tiered_abort_use_ts;-b {CMAKE_BINARY_DIR} -z"
    DEPENDS "WT_POSIX"
)
```
Again, I realize this might be out of the scope of this ticket. I figured it might be nice to properly put `tiered_abort` into testing so we can make sure it continues to work with CMake.

Reply:
@alisonfel and @jiechenbo the smoke test runs are effectively "commented out" for now in smoke.sh (by the call to exit 0).  If the new normal is to have cmake run these tests, it would be great to have it there, and commented out.  I suspect if you uncomment it just to test it, you may see it run, or see it crash, but that should at least let you know that the commented code is ready to go.

Reply:
I have done this! Thanks! Currently I commented out the whole segment. Let me know if this was something you guys were thinking about.